easybuild/easyconfigs/n/ncurses/ncurses-6.2-GCCcore-9.3.0.eb

```
@@ -0,0 +1,45 @@
+easyblock = 'ConfigureMake'
+
+name = 'ncurses'
+version = '6.2'
+
+homepage = 'https://www.gnu.org/software/ncurses/'
+
+description = """
+ The Ncurses (new curses) library is a free software emulation of curses in
+ System V Release 4.0, and more. It uses Terminfo format, supports pads and
+ color and multiple highlights and forms characters and function-key mapping,
+ and has all the other SYSV-curses enhancements over BSD Curses.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['30306e0c76e0f9f1f0de987cf1c82a5c21e1ce6568b9227f7da5b71cbea86c9d']
+
+builddependencies = [('binutils', '2.34')]
+
+local_common_configopts = "--with-shared --enable-overwrite --without-ada --enable-symlinks "
+configopts = [
+    # default build
+    local_common_configopts,
+    # the UTF-8 enabled version (ncursesw)
+    local_common_configopts + "--enable-ext-colors --enable-widec --includedir=%(installdir)s/include/ncursesw/",
+]
+
+local_libs = ["form", "menu", "ncurses", "panel"]
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ["captoinfo", "clear", "infocmp",
+                                     "infotocap",
+                                     "ncurses%(version_major)s-config",
+                                     "reset", "tabs", "tic", "toe", "tput",
+                                     "tset"]] +
```

Very minor change: I would reduce the list of bins to two lines
```suggestion
    'files': ['bin/%s' % x for x in ["captoinfo", "clear", "infocmp", "infotocap", "ncurses%(version_major)s-config",
                                     "reset", "tabs", "tic", "toe", "tput", "tset"]] +
```